test/Atmos/EDMF/closures/mixing_length.jl

```diff
@@ -91,9 +91,9 @@ function mixing_length(
     w_up = vuntuple(i -> up[i].ρaw / up[i].ρa, N_up)
     b = sum(
         ntuple(N_up) do i
-            a_up[i] * w_up[i] * δ[i] / env.a *
+            a_up[i] * w_up[i] * Δ[i]/max(up[i].ρaw,eps(FT)) / env.a *
             ((w_up[i] - env.w) * (w_up[i] - env.w) / 2 - tke_en) -
-            a_up[i] * w_up[i] * (w_up[i] - env.w) * εt[i] * env.w / env.a
+            a_up[i] * w_up[i] * (w_up[i] - env.w) * Et[i]/max(up[i].ρaw,eps(FT)) * env.w / env.a
```

Comment:
Same here, you are multiplying and dividing by `a_up[i] * w_up[i]`

Reply:
by definition εt[I]=Et[I]/up[i].ρaw

Reply:
Same as above, if you look at equation (19) you will see that we do not need it.